package.json

```diff
@@ -121,6 +121,7 @@
     "yaml-front-matter": "^4.0.0"
   },
   "devDependencies": {
+    "babel-plugin-lodash": "^3.3.4",
```

Comment:
I think this needs to be in `dependencies` as projects which use auspice as a dependency still need access to `[npx] auspice build` which uses babel.